test/SymmetricPluginManagerTest.php

```diff
@@ -44,6 +45,9 @@ public function testHas($symmetric)
     public function testGet($symmetric)
     {
         $plugin = new SymmetricPluginManager();
+        if (! extension_loaded($symmetric)) {
```

Comment:
the extension_loaded check can be placed before `new SymmetricPluginManager` creation.